create-image.sh

```diff
@@ -43,6 +43,19 @@ if [ ! "$(ls -A $JETSON_BUILD_DIR)" ]; then
         printf "\e[32mDownload L4T...       "
         wget -qO- $BSP | tar -jxpf - -C $JETSON_BUILD_DIR
         printf "[OK]\n"
+
+        # Fix nvidia's bugs in various BSP scripts
+        case "$BSP" in
+            *32.5*)
+                # Fix link dereferencing in 32.5 for xavier
+                sed -i 's/cp -f/cp -af/g' "$JETSON_BUILD_DIR/Linux_for_Tegra/tools/ota_tools/version_upgrade/ota_make_recovery_img_dtb.sh"
```

Comment:
The change here is to remove the `-a` flag? Please elaborate in the comment on why that change is needed possibly note what was breaking without this change.

Reply:
It adds "-a" flag because without it `cp` fails to copy anything that's a symlink and the whole recovery partition creation process fails. I have no idea how this ever passed any review/qa because it fails even when we use l4t example rootfs.